javascript/vale-a-pena-aprender-vanillajs--com-certeza.md

```diff
@@ -0,0 +1,98 @@
+[Artigo original: Is Vanilla JavaScript worth learning? Absolutely.](https://medium.freecodecamp.org/is-vanilla-javascript-worth-learning-absolutely-c2c67140ac34)
+
+---
+
+
+# Vale a pena aprender Vanilla JS? Claro!
+
+O propósito deste artigo é enfatizar a importância dos fundamentos JavaScript para todo programador front-end. Eu irei te dizer porque você deve ter um bom conhecimento do puro (Vanilla) JavaScript. Isto significa JavaScript sem nenhum framework ou biblioteca adicional.
+
+Eu também irei mencionar alguns recursos que me ajudaram a aprender o básico.
+
+Outra razão por trás desse post é que muitos aspirantes a desenvolvedor web tendem a não aprender os principais conceitos JavaScript como hoisting, closures, ou protótipos. Eles vão diretamente para os frameworks mais populares como React ou Angular 2. Eu irei te mostrar porque esta abordagem não é a mais conveniente.
+
+## Todo mundo quer desenvolvedores com conhecimento de ${insiraUmFrameworkPopularAqui}...
+
+Então, existe alguma razão pela qual você deveria se importar em aprender Vanilla JS?
+
+É difícil se tornar um(a) bom(a) desenvolvedor(a) sem conhecer o principal da linguagem.
+
+Assim como você precisa entender os princípios fundamentais da lei antes de poder entrar em uma área específica da lei. Esta comparação é puramente acidental. 😉
```

Comment:
Linkar `https://ideas.ataccama.com/i-stopped-being-a-lawyer-became-a-developer-and-its-awesome-5311e8d74882` em `Esta comparação`.

Seria uma boa traduzir esse tb? :P

Reply:
Certamente rs